### DISCUSSION TITLE
Bought a license, but it doesn't work

### DISCUSSION THREAD

**Discussion by BingThePing** [aclap-dev/video-downloadhelper#1439] [License & Purchase requests]
### What's your issue?

I wanted to download more than 1 video/hour, so a popup showed up saying I need a license. I bought it, however it did not activate because my browser version wasn't matching? I checked the email and it said that if I purchased the license on the browser that I intend to use it on then it should already be activated, but its not. I'm using opera GX and got the extension from the chrome store. When I checked settings it said "License is for Firefox but extension build is for Chrome". Can I get a refund or get switch the license somehow?
![image](https://github.com/user-attachments/assets/1f854c6a-979e-4438-ab16-e9147cacc92b)

### The 10 first digits (only 10) of your license number or email address

b62f91116a

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
We do not officially support Opera, that's probably why you have been lead to the wrong premium.
Do you want me to refund this one?
You can purchase Chrome premium from here: https://www.downloadhelper.net/convert?browser=chrome

**Reply by BingThePing** [reply]
yes

**Reply by jcfrog** [reply]
You have been refunded. Should be available in a few business days.

**Reply by BingThePing** [reply]
thanks, just to confirm, if i buy the chrome version it should activate right?

**Reply by jcfrog** [reply]
It should but once again, no guarantee

**Comment by peyotanchev** [comment]
The same problem
I can't use it on any browser
I want a refund and instructions on which browser to install the new license, Thanks in advance
peyotanchev@gmail.com

**Reply by jcfrog** [reply]
Sorry, your license is 3 months old, refund is only available the first month.
Your Chrome premium key is valid, why can't you use it on Chrome?

**Comment by Stringer63** [comment]
i bought a licence a while ago and its for Chrome now Chrome wont work on YouTube can mine be switched? Im Not re buying for Firefox

**Reply by jcfrog** [reply]
If you need to change your premium, you have one month to ask for a refund.

**Comment by daciusng** [comment]
I just purchase the premium license for Chrome, but it won't work for me to download a video from the site I want. I would like to ask for refund.

**Reply by daciusng** [reply]
What is the procedure for requesting refund?

**Reply by jcfrog** [reply]
hi, please send us an email at licenses@downloadhelper.net

**Reply by gimmickygyudon** [reply]
@jcfrog, hi how long it take to get refunded?

**Reply by jcfrog** [reply]
Stripe processes it, they say it can be up to 10 business days. Generally it's shorter than that but sometimes...
